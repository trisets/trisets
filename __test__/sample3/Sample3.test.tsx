import Sample3 from '@/pages/sample3';
import { render } from '@testing-library/react';
import { expect, test } from 'vitest';

test('sample2', () => {
  const { container } = render(<Sample3 />);
  expect(container.firstChild).toMatchSnapshot();
});
