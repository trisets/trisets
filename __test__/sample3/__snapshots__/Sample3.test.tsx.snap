// Vitest Snapshot v1

exports[`sample2 1`] = `
<h1>
  Page03 desuyo
</h1>
`;
